### PR TITLE
Fix syntax error in type annotation

### PR DESCRIPTION
**Pull request by daikikatsuragawa**
Signed-off-by: Daiki Katsuragawa <50144563+daikikatsuragawa@users.noreply.github.com>

When I executed the following command ...

```sh
mypy --ignore-missing-imports --warn-unreachable core/
```

The following error was detected.

```sh
core/src/autogluon/core/utils/utils.py:368: error: Syntax error in type annotation  [syntax]
```

When a function returns more than one value, the values are represented as tuples.

### CODE REVIEW

**Comment by daikikatsuragawa** [comment]
For example, I get the following error with the following task.

https://github.com/awslabs/autogluon/actions/runs/3455985811/jobs/5768417443

```sh
Error: 🚨 Insuffient Permissions! daikikatsuragawa does not have write permissions
```

What should I do?

File: core/src/autogluon/core/utils/utils.py [hunk 1]
```diff
@@ -365,7 +365,7 @@ def generate_train_test_split(X: DataFrame,
                               problem_type: str,
                               test_size: float = 0.1,
                               random_state=0,
-                              min_cls_count_train=1) -> (DataFrame, DataFrame, Series, Series):
+                              min_cls_count_train=1) -> Tuple(DataFrame, DataFrame, Series, Series):
```
Comment (yinweisu) [inline core/src/autogluon/core/utils/utils.py:368]: I believe it should be `Tuple[]`
Ditto for the rest

**Review by yinweisu — CHANGES_REQUESTED** [review]
Thanks for the PR. We probably won't merge this in right now as we are in code freeze for 0.6 release

File: core/src/autogluon/core/utils/utils.py (hunk 1 above)
Comment (daikikatsuragawa) [reply core/src/autogluon/core/utils/utils.py]: @yinweisu 

Thank you for pointing that out. I agree with your comment. I have fix it!

**Comment by daikikatsuragawa** [comment]
[Merge branch 'master' into master](https://github.com/awslabs/autogluon/pull/2376/commits/bb85eae62786f78a4d0f941044acdf10caefac46)

This is a response to conflicts.

**Review by Innixma — APPROVED** [review]
LGTM, thanks for the contribution!

**Pull request merged**